python_builtin_functions.py

```diff
@@ -4,86 +4,170 @@ def myabs(number):
         return -number
     return number
 
+
 def myall(iterable):
     """ Implementation of built-in `all` function """
-    pass
+    for item in iterable:
+        if not item:
+            return False
+    return True
 
 
 def myany(iterable):
     """ Implementation of built-in `any` function """
-    pass
+    for item in iterable:
+        if item:
+            return True
+    return False
 
 
 def mylen(s):
     """ Implementation of built-in `len` function """
-    pass
+    i=0
+    for _ in s:
+        i+=1
+    return i
 
 
 def mysum(iterable):
     """ Implementation of built-in `sum` function """
-    pass
+    i=0
+    for x in iterable:
+        i+=x
+    return i
 
 
 def mymax(iterable):
     """ Implementation of built-in `max` function """
-    pass
+    i=0
+    for x in iterable:
+        if x>i:
+            i=x
+        else:
```

Comment:
`else` block is not required as there is only one `continue` statement and no other statements are present

Reply:
removed else block